services/QuillLMS/app/services/learn_worlds_integration/user_tags_request.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module LearnWorldsIntegration
+  class UserTagsRequest < UserRequest
+
+    attr_reader :external_id, :tags
+
+    def endpoint
+      "#{USER_TAGS_ENDPOINT}/#{external_id}"
+    end
+
+    def initialize(external_id, tags)
+      raise ArgumentError, "Invalid tags: #{tags}" unless tags.respond_to?(:each)
+
+      @external_id = external_id
+      @tags = tags
+    end
+
+    def run = HTTParty.put(endpoint, body: data.to_json, headers: headers)
```

Comment:
Again, you can shortcut the `headers:` part of this since the value has the same name as the key.  So: `HTTParty.put(endpoint, body: data.to_json, headers:)`

Additional thought: in order to take advantage of that, thoughts on renaming `def data` to `def body` and including the `to_json` call in that definition?  Feel free to not do that, but I do like naming consistency when it's easy.